global/global.ts

```diff
@@ -1,2 +1,172 @@
-import 'allocator/arena'
-export { memory }
+import { BigDecimal, TypedMapEntry, JSONValue, Value, Entity, TypedMap, Result, Wrapped } from '../index'
+import { ethereum } from '../chain/ethereum'
+
+export enum TypeId {
```

Comment:
The order of this enum is up to change, also it's an internal thing, even if we change it won't affect users.

The thing is to keep `graph-node` with the same values.